internal/pritunl/user.go

```diff
@@ -26,10 +29,43 @@ type User struct {
 	OtpAuth         bool                     `json:"otp_auth,omitempty"`
 	DeviceAuth      bool                     `json:"device_auth,omitempty"`
 	Organization    string                   `json:"organization,omitempty"`
+	Pin             Pin                      `json:"pin,omitempty"`
 }
 
 type PortForwarding struct {
 	Dport    string `json:"dport"`
 	Protocol string `json:"protocol"`
 	Port     string `json:"port"`
 }
+
+type Pin struct {
+	IsSet  bool
```

Comment:
I don't see any read of the field `IsSet`, only two write calls. Do you really need it?

Reply:
I don't think so. This was added just to keep the functionality previously provided by the `Pin bool` field, which I understood was to be kept.